diff --git a/ext/opus/meson.build b/ext/opus/meson.build
index 13fd5d485..6f9f9f10b 100644
--- a/ext/opus/meson.build
+++ b/ext/opus/meson.build
@@ -6,7 +6,7 @@ opus_sources = [
   'gstopusheader.c',
 ]
 
-opus_dep = dependency('opus-avx', version : '>= 0.9.4', required : get_option('opus'), static : true)
+opus_dep = dependency('opus', version : '>= 0.9.4', required : get_option('opus'), static : false)
 
 if opus_dep.found()
   gstopus = library('gstopus',
